Mathlib/CategoryTheory/FiberedCategory/Fibered.lean

```diff
@@ -72,4 +93,95 @@ lemma pullbackObj_proj : p.obj (pullbackObj ha f) = R :=
 
 end IsPreFibered
 
+namespace IsFibered
+
+open IsCartesian IsPreFibered
+
+/-- In a fibered category, any cartesian morphism is strongly cartesian. -/
+instance isStronglyCartesian_of_isCartesian (p : 𝒳 ⥤ 𝒮) [p.IsFibered] {R S : 𝒮} (f : R ⟶ S)
+    {a b : 𝒳} (φ : a ⟶ b) [p.IsCartesian f φ] : p.IsStronglyCartesian f φ where
+  universal_property' := by
+    intro a' g φ' hφ'
```

Comment:
```suggestion
  universal_property' g φ' hφ' := by
```
(Note that if we absolutely had to give a name to the first implicit variable, we could also use the syntax `universal_property' {a'} g φ' hφ' :=`.